geometry_msgs/Point point
string mission_name
bool doMission
